Meadowlark-js/packages/meadowlark-utilities/src/Environment.ts

```diff
@@ -61,7 +62,7 @@ export function getValueFromEnvironment<T>(
 ): T {
   const cached = getFromCache<T>(key);
 
-  if (cached) {
+  if (cached !== undefined) {
```

Comment:
Found a bug. If the cached value were the boolean `false`, then `if (cached)` was false, when it should be true.